addons/spreadsheet/static/src/data.json

```diff
@@ -0,0 +1,141 @@
+{
+  "version": 12,
+  "sheets": [
+    {
+      "id": "Sheet1",
+      "name": "Sheet1",
+      "figures": [
+        {
+          "id": "bb8b5846-b950-4541-8bc0-8dbe30e14b63",
+          "x": 10,
+          "y": 10,
+          "width": 536,
+          "height": 335,
+          "tag": "chart",
+          "data": {
+            "title": "Opportunities",
+            "background": "#FFFFFF",
+            "legendPosition": "top",
+            "definition": {
+              "id": "bb8b5846-b950-4541-8bc0-8dbe30e14b63",
+              "groupBy": [
+                "stage_id",
+                "country_id"
+              ],
+              "measure": "__count",
+              "order": "ASC",
+              "model": "crm.lead",
+              "context": {},
+              "domain": []
+            },
+            "type": "odoo_bar",
+            "verticalAxisPosition": "left",
+            "stacked": true,
+            "fieldMatching": {
+              "5efe8c20-c0a2-4ad5-bff6-379605a6aea0": {
+                "chain": "create_date",
+                "offset": 0
+              },
+              "e7bf233b-d9e4-4159-865e-77edd1195fdb": {
+                "chain": "country_id"
+              }
+            }
+          }
+        }
+      ]
+    }
+  ],
+  "odooVersion": 5,
+  "globalFilters": [
+    {
+      "id": "5efe8c20-c0a2-4ad5-bff6-379605a6aea0",
+      "type": "date",
+      "label": "date filter",
+      "defaultValue": {},
+      "rangeType": "year",
+      "defaultsToCurrentPeriod": false
+    },
+    {
+      "id": "e7bf233b-d9e4-4159-865e-77edd1195fdb",
+      "type": "relation",
+      "label": "country",
+      "model": "res.country",
+      "defaultValue": [],
+      "defaultValueDisplayNames": [],
+      "rangeType": "year",
+      "defaultsToCurrentPeriod": false
+    }
+  ],
+  "pivots": {
+    "1": {
+      "colGroupBys": [
+        "stage_id"
+      ],
+      "context": {},
+      "domain": [],
+      "id": "1",
+      "measures": ["expected_revenue", "recurring_revenue_monthly"],
+      "model": "crm.lead",
+      "rowGroupBys": [
+        "create_date:month"
+      ],
+      "name": "Pipeline Analysis by Stage",
+      "orderBy": { "field": "expected_revenue", "asc": true},
```

Comment:
alternatives that exists in the code base
1. `"orderBy": { "field": "expected_revenue", "asc": true}`
2. `"orderBy": "expected_revenue ASC"`
3. `"orderBy": { "field": "expected_revenue", "order": "asc"}`

I like the simplicity and readability of 2. but there's additional logic to "parse" the string (duplicated in upgrade scripts)

Reply:
1. is the ORM api, so it's probably the way to go

Reply:
Actually it's `{name: 'bar', asc: false}` which doesn't look good.

Reply:
For me `2` is ok, we already have a parsing process for group bys with a date (`create_date:day`). Either we go for `2`, either we change group by to something like `{ field: "create_date", group: "day" }`.
Anyway, at runtime, it should be great to not have to parse each time we need the information

Reply:
Actually we need the "groupId". It's used to sort values on a specific column. Let's take the default CRM pivot, it's different to sort by `expected_revenue` on the `total` or when `stage_id` = `Qualified`.